packages/capabilities/src/types.ts

```diff
@@ -271,13 +283,21 @@ export interface StoreListItem {
   link: UnknownLink
   size: number
   origin?: UnknownLink
+  insertedAt: string
 }
 
-export interface UploadAddSuccess {
+export interface UploadListItem {
   root: UnknownLink
   shards?: CARLink[]
+  insertedAt: string
+  updatedAt: string
```

Comment:
Add insertedAt and updatedAt props to UploadListItem. We want to share a timestamp with the user about when this upload was added, and these are the props we have. `updatedAt` currently means "when did we last add a shard to this upload", while `insertedAt` means "when did you first add a given root cid as an upload to this space."